src/Compilers/VisualBasic/Portable/Compilation/SyntaxTreeSemanticModel.vb

```diff
@@ -135,26 +135,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         ' PERF: These shared variables avoid repeated allocation of Func(Of Binder, MemberSemanticModel) in GetMemberSemanticModel
-        Private Shared ReadOnly s_methodBodySemanticModelCreator As Func(Of Tuple(Of Binder, Boolean), MemberSemanticModel) = Function(key As Tuple(Of Binder, Boolean)) MethodBodySemanticModel.Create(DirectCast(key.Item1, SubOrFunctionBodyBinder), key.Item2)
-        Private Shared ReadOnly s_initializerSemanticModelCreator As Func(Of Tuple(Of Binder, Boolean), MemberSemanticModel) = Function(key As Tuple(Of Binder, Boolean)) InitializerSemanticModel.Create(DirectCast(key.Item1, DeclarationInitializerBinder), key.Item2)
-        Private Shared ReadOnly s_attributeSemanticModelCreator As Func(Of Tuple(Of Binder, Boolean), MemberSemanticModel) = Function(key As Tuple(Of Binder, Boolean)) AttributeSemanticModel.Create(DirectCast(key.Item1, AttributeBinder), key.Item2)
+        Private Shared ReadOnly s_methodBodySemanticModelCreator As Func(Of Tuple(Of SyntaxTreeSemanticModel, Binder, Boolean), MemberSemanticModel) = Function(key As Tuple(Of SyntaxTreeSemanticModel, Binder, Boolean)) MethodBodySemanticModel.Create(key.Item1, DirectCast(key.Item2, SubOrFunctionBodyBinder), key.Item3)
```

Comment:
>SyntaxTreeSemanticModel [](start = 85, length = 23)

Rather than adding this instance to the key, it would probably be better to change these fields to instance fields and pass ```Me``` to the ```Create``` calls. The "s" prefix would have to be removed from the names of the fields as well.